core-plugins/pom.xml

```diff
@@ -221,6 +231,12 @@
       <artifactId>format-orc</artifactId>
       <version>${project.version}</version>
       <scope>test</scope>
+      <exclusions>
```

Comment:
nit: If this conflicts with another dependency, can we add a brief comment here on which one?